apps/odo/mix.exs

```diff
@@ -30,7 +30,7 @@ defmodule Odo.MixProject do
       {:credo, "~> 1.0", only: :dev, runtime: false},
       {:distillery, "~> 2.1"},
       {:ex_aws, "~> 2.1"},
-      {:ex_aws_s3, "~> 2.0"},
+      {:ex_aws_s3, "~> 2.0", [env: :prod, git: "https://github.com/ex-aws/ex_aws_s3", ref: "6b9fdac73b62dee14bffb939965742f2576f2a7b"]},
```

Comment:
What happened here?  Do we need this very specific version that isn't available from hex?

Reply:
Yeah, it is needed by discovery_api in order to use streaming. Still pending release. I'll add a comment to make that clear.

Reply:
Yeah, also, this is the one that overflows cowboy when you try to stream hosted files, btw